src/pages/registration/SignUpWorkflow/CartPreview.tsx

```diff
@@ -0,0 +1,182 @@
+import * as React from "react";
+import Typography from "@material-ui/core/Typography";
+import Grid from '@material-ui/core/Grid';
+import Paper from "@material-ui/core/Paper";
+import Divider from "@material-ui/core/Divider";
+
+import { isApiErrorResponse, listBillingDiscounts, } from "makerspace-ts-api-client";
+import { discountParam, invoiceOptionParam } from '../MembershipOptions';
+import { useSearchQuery, useSetSearchQuery } from "hooks/useSearchQuery";
+import { useMembershipOptions } from "hooks/useMembershipOptions";
+import useReadTransaction from "ui/hooks/useReadTransaction";
+import { noneInvoiceOption } from "ui/membership/MembershipSelectForm";
+import { numberAsCurrency } from "ui/utils/numberAsCurrency";
+import { TextInput } from "components/Form/inputs/TextInput";
+import { CheckboxInput } from "components/Form/inputs/CheckboxInput";
+import { FormField } from "components/Form/FormField";
+import KeyValueItem from "ui/common/KeyValueItem";
+
+interface Props {
+  readOnly?: boolean;
+}
+
+const ssmDiscount = "MMS_10%";
```

Comment:
Should this be part of membership options? For that matter, should membership options also fetch discounts?